resources/kcp/values.yaml

```
@@ -16,6 +16,9 @@ global:
     kyma_environments_cleanup_job:
       dir:
       version: "PR-83"
```

please also bump cleanup job image because it was modified in this PR

```suggestion
      version: "PR-131"
```